server/services/datacommons.py

```diff
@@ -177,19 +177,16 @@ def v2observation(select, entity, variable):
   Args:
     select: A list of select props.
     entity: A dict in the form of {'dcids':, 'expression':}
-    variable: A dict in the form of {'dcids':, 'expression':}
+    variable: A dict in the form of {'dcids':, 'expression':}    
 
   """
+  req = {'select': select}
   if 'dcids' in entity:
-    entity['dcids'].sort()
+    req['entity'] = {'dcids': sorted(entity['dcids'])}
```

Comment:
'entity' would not be set if it is {'expression': xxx}. Can just do:

entity['dcids'] = sorted(entity['dcids'])

?

Reply:
good point.